app/src/main/java/com/jerboa/db/repository/AppSettingsRepository.kt

```diff
@@ -30,29 +19,12 @@ class AppSettingsRepository(
     }
 
     @WorkerThread
-    suspend fun markChangelogViewed() {
-        appSettingsDao.markChangelogViewed()
+    suspend fun updateLastVersionCodeViewed(versionCode: Int) {
+        appSettingsDao.updateLastVersionCode(versionCode)
     }
 
     @WorkerThread
     suspend fun updatePostViewMode(postViewMode: Int) {
         appSettingsDao.updatePostViewMode(postViewMode)
     }
-
-    @WorkerThread
-    suspend fun updateChangelog() {
-        withContext(Dispatchers.IO) {
```

Comment:
I removed this `withContext` when I moved this to the viewmodel. It didn't seem to have any effect.